docs/2.Basics/CLI Command Reference.md

```diff
@@ -62,4 +62,6 @@ nav_order: 2
 | `--openai-api-key`                                                                                                                                                                                                                                                                                                                                                         | Add an OpenAI API key to enhance finding guidelines by sending violated policies and resource code to OpenAI to request remediation guidance. This will use your OpenAI credits. Set your number of findings that will receive enhanced guidelines using CKV_OPENAI_MAX_FINDINGS                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `--tool-name`                                                                                                                                                                                                                                                                                                                                                         | Add a custom tool name to change the tool name field, this is especially useful for outputting results in SARIF format for upload to Github Code Scanning                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
I would call it `custom-tool-name` to be more precise on its meaning

Reply:
That makes sense, I have implemented this change